functions/archive

```diff
@@ -17,12 +17,13 @@ fi
 case "${1}" in
   (*.tar.gz|*.tgz) tar -cvzf "${@}" ;;
   (*.tar.bz|*.tar.bz2|*.tbz|*.tbz2) tar -cvjf "${@}" ;;
-  (*.tar.xz|*.txz) tar -J --help &>/dev/null && tar -cvJf "${@}" ;;
-  (*.tar.lzma|*.tlz) tar --lzma --help &>/dev/null && tar --lzma -cvf "${@}" ;;
+  (*.tar.xz|*.txz) tar -J --help &>/dev/null && env XZ_OPT=-T0 tar -cvJf "${@}" ;;
+  (*.tar.lzma|*.tlz) tar --lzma --help &>/dev/null && env XZ_OPT=-T0 tar --lzma -cvf "${@}" ;;
   (*.tar) tar -cvf "${@}" ;;
   (*.zip) zip -r "${@}" ;;
   (*.rar) rar a "${@}" ;;
   (*.7z) 7za a "${@}" ;;
+  (*.zst) zstd -c -T0 "${@}" -o "${1}" ;;
```

Comment:
You might want to offset the args list with `${@:2}`, so `${1}` in not included there.

Also, please include zst in the README.md.

Reply:
Of course. Thank you for the suggested fix.